test/TestPercentageMath.sol

```diff
@@ -64,6 +82,37 @@ contract TestPercentageMath is Test {
         assertEq(PercentageMath.percentDiv(x, y), PercentageMathRef.percentDiv(x, y));
     }
 
+    function testPercentAvg(
+        uint256 x,
+        uint256 y,
+        uint16 percentage
+    ) public {
+        vm.assume(percentage <= PERCENTAGE_FACTOR);
```

Comment:
We can remove this line, no ?

Reply:
If this line gets removed then `percentage` is no longer bounded and can lead to a revert when passed to `percentAvg`

Reply:
Isn't it a duplicate of one of the checks of the line coming just after ?
```solidity
vm.assume(
            percentage < PERCENTAGE_FACTOR && x <= MAX_UINT256_MINUS_HALF_PERCENTAGE / (PERCENTAGE_FACTOR - percentage)
        );
```

Reply:
oh you just highlighted a mistake: this line should be:
```solidity
vm.assume(
            percentage == PERCENTAGE_FACTOR || x <= MAX_UINT256_MINUS_HALF_PERCENTAGE / (PERCENTAGE_FACTOR - percentage)
        );
```